libs/components/modals/src/lib/modules/modal/modal.component.html

```diff
@@ -59,7 +59,7 @@
     <div
       class="sky-modal-content sky-padding-even-large"
       role="region"
-      tabindex="0"
+      tabindex="-1"
```

Comment:
does anyone know why we had this focusable? Level Access dinged us because there is nothing to be interacted with